### ISSUE TITLE
Trailing commas

### ISSUE THREAD

**Issue by SheepTester** [nbuilding/N-lang#76]
The current syntax does not allow for trailing commas.

```ts
let personDecoder = D.map2(
	person,
	D.field("name", D.str),
	D.field("height", D.float),
)
```

The benefit of consistently using trailing commas is that it does not result in the following diffs:

```diff
  let coolList: [
    "Billy",
    "Bob",
-   "Joe"
+   "Joe",
+   "Jilly"
  ]
```

**Comment by Ashvin-Ranjan** [comment]
It is really easy to implement just change
https://github.com/nbuilding/N-lang/blob/6877ee22440b9feb8dd18eb179835c77e00ca2bf/python/syntax.lark#L52-L53
to
```
tupleval: "(" expression ("," expression)* (",")? ")" 
listval: "[" (expression ("," expression)* (",")?)? "]" 
```
this may not work completely but this is the main thing

**Comment by SheepTester** [comment]
epic

**Comment by Ashvin-Ranjan** [comment]
Fixed as of Pull Request #83
